docs/v1.1.0/schema.md

```diff
@@ -0,0 +1,216 @@
+
+## 5. Annotation Metadata
+
+### 5.1 Annotation
+Describe the annotations of the CryoET dataset, including annotators, annotation objects, annotation methods etc.
+
+| Data elements                 | metadata key 1           | Sub-elements                      | metadata key 2              | Value type | Unit | Default | Requirement                                                           | Controlled vocabularies                                           | Description                                                                                                                                                                                                                                                                                                                        |
+|-------------------------------|--------------------------|-----------------------------------|-----------------------------|------------|:----:|:-------:|-----------------------------------------------------------------------|-------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Annotation file name          | ???                      | N/A                               |                             | STRING     |  --  |   --    | MUST                                                                  |                                                                   | Name of an annotation file. We adopt the following naming convention: tilt_experiment.name+object.name+annotation_instance.name+version.number. There could be multiple annotations by multiple annotators for the same CryoET datasets (tilt series and tomograms), each annotation file is considered an ‘annotation instance’). |
+| Authors                       | authors                  | ORCID                             | ORCID                       | STRING     |  --  |   --    | RECOMMENDED                                                           | ORCID                                                             | A unique, persistent identifier for the annotator, provided by ORCID                                                                                                                                                                                                                                                               |
```

Comment:
'ORCID' is the only metadata key that is capitalized. Should probably be lower case in the future.